llvm/include/llvm/ProfileData/Coverage/CoverageMapping.h

```diff
@@ -813,7 +813,7 @@ template <class FuncRecordTy, support::endianness Endian>
 Error getFuncNameViaRef(const FuncRecordTy *Record,
                         InstrProfSymtab &ProfileNames, StringRef &FuncName) {
   uint64_t NameRef = getFuncNameRef<FuncRecordTy, Endian>(Record);
-  FuncName = ProfileNames.getFuncName(NameRef);
+  FuncName = ProfileNames.getValueName(NameRef);
```

Comment:
nit: make the name clearer: getFuncOrVarName().

Reply:
done.